charts/backend-service/values.yaml

```
@@ -94,6 +94,12 @@ tyk:
     #     timeoutSeconds: 1000
   # defaultTimeout: 30 # This setting is only to be modified after consulting the platform team. Disagreed changes will be reverted unquestioned.
 
+  # blockList allows you to block specific paths and methods
```

Please use valid helm-docs for automatic documentation
```suggestion
  # -- blockList allows you to block specific paths and methods
```